docker_plugin/tests/blueprint/test_pull.yaml

```diff
@@ -38,6 +38,13 @@ node_templates:
               stdin_open: true
               tty: true
               command: /bin/sleep 30
+            daemon_client:
```

Comment:
We assume that user may be that stupid to override plugin default value definitions.